crates/verify/src/verify.rs

```diff
@@ -357,9 +357,7 @@ impl VerifyArgs {
                 }
 
                 let profile = profiles.into_iter().next().unwrap().to_owned();
-                let settings = cache.profiles.get(&profile).expect("must be present");
-
-                settings
+                cache.profiles.get(&profile).expect("must be present")
```

Comment:
clippy lint rule now enforces `returning the result of a `let` binding from a block` preferring direct returns where possible